draft/implementation-notes/index.html

```diff
@@ -538,6 +538,136 @@ <h2>File Purging</h2>
       </p>
     </section>
 
+    <section>
+      <h2>Migrating to a New Digest Algorithm</h2>
+      <p>
+        Over time new digest algorithms are developed to increase security and address vulnerabilities in existing
+        algorithms. It may become desirable to migrate an object to use a new algorithm while retaining
+        <a href="#version-immutability">Version Immutability</a>. OCFL supports this through the creation of a new
+        version with a new <code>digestAlgorithm</code> that either retains the same object content or is combined with
+        a content update.
+      </p>
+      <p>
+        Consider an example OCFL object created using the <code>digestAlgorithm</code> <code>sha256</code> with content
```

Comment:
I'd suggest changing the sentence to:

> Consider an example OCFL object where the <code>digestAlgorithm</code> in the inventory.json is <code>sha256</code> and the OCFL object contains a single file (<code>file.txt</code>).

It reads a bit more cleanly.

Reply:
Yes, that is clearer!